consensus/src/main/java/org/ethereum/beacon/consensus/SpecHelpers.java

```diff
@@ -12,11 +12,14 @@
 import java.util.List;
```

Comment:
May be let's finally decide on style enforcement and if decided to use strict format then reformat all the classes separately?